core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-hint-modal.component.ts

```diff
@@ -73,7 +73,29 @@ export class AddHintModalComponent
   }
 
   isHintLengthExceeded(tmpHint: string): boolean {
-    return Boolean(tmpHint.length > 500);
+    // The variable chars stores the remaining characters by
+    // removing the html tags from tmpHint.
+    let charRgx = /(?<=>)([^(\n)]*?)(?=<)/g;
```

Comment:
If this is a constant, let's make sure to follow the pattern in the codebase used for constants (see examples in the same file). Also, for naming, would TEXT_NODE_REGEX or similar be clearer?